### DISCUSSION TITLE
Trigger email notifications through api

### DISCUSSION THREAD

**Discussion by owl-se** [reportportal/reportportal#1251] [Q&A] [answered]
Hey all.

I've been wondering is it possible to trigger notifications through api calls?
I saw "/v1/project/{projectName}/notification" but it rather for configuration purposes.

**Comment by evjlobanova** [comment, marked as answer]
There is no such option, right now, notification is sent only on launch finish

**Reply by evjlobanova** [reply]
We are planning to add this functionality in the few next version
